lib/mermaid.jsx

```diff
@@ -55,15 +55,14 @@ export default class Mermaid extends React.Component {
       </div>
     )
   }
-  renderDiagram = () => {
+  renderDiagram = async () => {
     const { children } = this.props
     const [code] = children || []
     try {
       this.cleanupMermaidDiv()
       if (typeof code === 'string') {
-        mermaidAPI.render(this.mermaidId, code, svg =>
-          this.setState({ svg, error: null })
-        )
+        const { svg } = await mermaid.render(this.mermaidId, code)
+        this.setState({ svg, error: null })
       }
     } catch (e) {
       this.setState({ error: e, svg: '' })
```

Comment:
also found that there is an error: `TypeError: Cannot read properties of null (reading 'firstChild')` on every initial rendering if I add `console.error` at this line.

![Screenshot 2024-01-05 at 16 20 15](https://github.com/inkdropapp/inkdrop-mermaid/assets/1332805/5a620469-1338-47e9-aafd-77509b52b340)

It looks like calling `this.cleanupMermaidDiv()` is no longer needed in `renderDiagram` on v10, which resolves this error.
Can you please check it?

Reply:
I'm not getting the error but also noticed this new version implements clean up internally, removed the method `cleanupMermaidDiv`

Reply:
yes, it's resolved!